zips/zip-0230.rst

```diff
@@ -160,8 +160,8 @@ Transaction Format
 +-----------------------------+------------------------------+------------------------------------------------+---------------------------------------------------------------------+
 | varies                      |``issuerLength``              |``compactSize``                                 |The length of the issuer identifier of the issuer.                   |
 +-----------------------------+------------------------------+------------------------------------------------+---------------------------------------------------------------------+
-| varies                      |``issuer``                    |``byte[issuerLength]``                          |The issuer identifier of the issuer, used to validate the            |
-|                             |                              |                                                |signature.                                                           |
+| varies                      |``issuer``                    |``byte[issuerLength]``                          |The issuer identifier of the issuer, equal to the issuance validating|
```

Comment:
"equal to the issuance validating" issuance validating what?

Reply:
meant to write `key` on the next line, missed that. Fixed it.